src/applications/vaos/services/appointment/index.js

```diff
@@ -677,3 +678,120 @@ export async function cancelAppointment({ appointment }) {
+        .format('h:mm a')}`,
+      phone: formatFacilityPhone(facility),
+      additionalText: [
+        'Sign in to VA.gov to get details about this appointment',
+      ],
+    };
+  } else if (isInPersonVAAppointment) {
+    data = {
+      summary: `Appointment at ${facility?.name}`,
+      location: formatFacilityAddress(facility),
+      text: `You have a health care appointment at ${facility?.name}`,
+      phone: formatFacilityPhone(facility),
+      additionalText: [
+        'Sign in to VA.gov to get details about this appointment',
+      ],
+    };
+  } else if (isCommunityCare) {
+    const { providerName, practiceName } =
+      appointment.communityCareProvider || {};
+
+    data = {
+      summary: `Appointment at ${providerName || practiceName}`,
+      providerName: `${providerName || practiceName}`,
+      location: `${formatFacilityAddress(appointment.communityCareProvider)}`,
+      text:
+        'You have a health care appointment with a community care provider. Please don’t go to your local VA health facility.',
+      phone: `${formatFacilityPhone(appointment.communityCareProvider)}`,
+      additionalText: [
+        'Sign in to VA.gov to get details about this appointment',
+      ],
+    };
+  } else if (isVideo) {
+    const providerName = appointment.videoData?.providers
+      ? appointment.videoData.providers[0]?.display
+      : '';
+
+    if (isHome) {
+      data = {
+        summary: 'VA Video Connect appointment',
+        text:
+          'You can join this meeting up to 30 minutes before the start time.',
+        location: 'VA Video Connect at home',
+        additionalText: ['Sign in to VA.gov to join this meeting'],
+      };
+    } else if (isAtlas) {
+      const { atlasLocation } = appointment.videoData;
+
+      if (atlasLocation?.address) {
+        data = {
+          // TODO: This doesn't seem right
+          summary: `VA Video Connect appointment at an ATLAS facility`,
+          // providerName: `${firstName} ${lastName}`,
+          location: formatFacilityAddress(atlasLocation),
+          text: 'Join this video meeting from this ATLAS (non-VA) location:',
+          additionalText: [
+            `Your appointment code is ${
+              appointment.videoData.atlasConfirmationCode
+            }. Use this code to find your appointment on the computer at ${providerName}.`,
+            `You'll be meeting with ${providerName}`,
+          ],
+        };
+      }
+    } else if (videoKind === VIDEO_TYPES.clinic) {
+      data = {
+        summary: `VA Video Connect appointment at ${
+          appointment.location.clinicName
+        }`,
+        providerName: appointment.location.clinicName,
+        location: formatFacilityAddress(facility),
+        text: 'You need to join this video meeting from:',
+        phone: `${formatFacilityPhone(facility)}`,
+        additionalText: [
+          `You’ll be meeting with ${appointment.location.clinicName}`, // TODO: Verify this
```

Comment:
I think this will read awkwardly for a lot of clinics, which might be named something like "Green team clinic 1." If we have provider information, it'll be in the same spot as for other video appointments.

Reply:
Done.